### PR TITLE
Fix paste text mess up style and Add support copy/cut block text.

### PR DESCRIPTION
**Pull request by TheHeBoy**
- Fix paste text mess up style 
- Add support copy/cut block text.

Demo:

https://github.com/singerdmx/flutter-quill/assets/49644098/808cd35e-8002-4edc-871f-db35cdebbf46

Issue:#1313

### CODE REVIEW

**Comment by singerdmx** [comment]
info • The import of '../../models/documents/document.dart' is unnecessary because all of the used elements are also provided by the import of '../../../flutter_quill.dart' • lib/src/widgets/raw_editor/raw_editor_state_selection_delegate_mixin.dart:7:8 • unnecessary_import
   info • The import of '../../models/documents/nodes/embeddable.dart' is unnecessary because all of the used elements are also provided by the import of '../../../flutter_quill.dart' • lib/src/widgets/raw_editor/raw_editor_state_selection_delegate_mixin.dart:8:8 • unnecessary_import
   info • The import of '../../models/documents/nodes/leaf.dart' is unnecessary because all of the used elements are also provided by the import of '../../../flutter_quill.dart' • lib/src/widgets/raw_editor/raw_editor_state_selection_delegate_mixin.dart:9:8 • unnecessary_import
   info • The import of '../editor.dart' is unnecessary because all of the used elements are also provided by the import of '../../../flutter_quill.dart' • lib/src/widgets/raw_editor/raw_editor_state_selection_delegate_mixin.dart:[11](https://github.com/singerdmx/flutter-quill/actions/runs/5628286512/job/15251972954?pr=1321#step:7:12):8 • unnecessary_import

**Comment by singerdmx** [comment]
Please don't use  '../../../flutter_quill.dart'

File: lib/src/widgets/raw_editor/raw_editor_state_selection_delegate_mixin.dart [hunk 1]
```diff
@@ -3,6 +3,7 @@ import 'dart:math' as math;
 import 'package:flutter/rendering.dart';
 import 'package:flutter/widgets.dart';
 
+import '../../../flutter_quill.dart';
```
Comment (singerdmx) [inline lib/src/widgets/raw_editor/raw_editor_state_selection_delegate_mixin.dart:6]: Please don't use  '../../../flutter_quill.dart'

**Comment by TheHeBoy** [comment]
I have a question: If I paste to a line with the style, apply the original style or style of paste position?

**Comment by singerdmx** [comment]
Try it in word and we should be the same

**Comment by TheHeBoy** [comment]
Ok, I will again pr resolve this question.

**Comment by TheHeBoy** [comment]
Do I need to update changeLog about this bug?

**Comment by singerdmx** [comment]
Yes

**Pull request merged**